src/Stryker.Core/Stryker.Core.UnitTest/MutantFilters/ExcludeLinqExpressionFilterTests.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+using Shouldly;
+using Stryker.Core.MutantFilters;
+using Stryker.Core.Mutants;
+using Stryker.Core.Mutators;
+using Stryker.Core.Options;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.MutantFilters
+{
+    public class ExcludeLinqExpressionFilterTests : TestBase
+    {
+
+        [Fact]
+        public void ShouldHaveDisplayName()
+        {
+            // Arrange
+            var sut = new ExcludeLinqExpressionFilter();
+
+            // Assert
+            sut.DisplayName.ShouldBe("linq expression filter");
+        }
+
+        [Theory]
+        [InlineData(LinqExpression.FirstOrDefault)]
+        [InlineData(LinqExpression.First)]
+        [InlineData(LinqExpression.SingleOrDefault)]
+        [InlineData(LinqExpression.Single)]
+        [InlineData(LinqExpression.Last)]
+        [InlineData(LinqExpression.All)]
+        [InlineData(LinqExpression.Any)]
+        [InlineData(LinqExpression.Skip)]
+        [InlineData(LinqExpression.Take)]
+        [InlineData(LinqExpression.SkipWhile)]
+        [InlineData(LinqExpression.TakeWhile)]
+        [InlineData(LinqExpression.Min)]
+        [InlineData(LinqExpression.Max)]
+        [InlineData(LinqExpression.Sum)]
+        [InlineData(LinqExpression.Average)]
+        [InlineData(LinqExpression.OrderBy)]
+        [InlineData(LinqExpression.OrderByDescending)]
+        [InlineData(LinqExpression.ThenBy)]
+        [InlineData(LinqExpression.ThenByDescending)]
+        [InlineData(LinqExpression.Reverse)]
+        [InlineData(LinqExpression.AsEnumerable)]
+        [InlineData(LinqExpression.Union)]
+        [InlineData(LinqExpression.Intersect)]
+        [InlineData(LinqExpression.Concat)]
+        [InlineData(LinqExpression.Except)]
+        public void ShouldRemoveLinqExpression(LinqExpression exp)
+        {
+            // Arrange
+            var target = new LinqMutator();
+
+            var expression = GenerateExpressions(exp.ToString());
+
+            var result = target.ApplyMutations(expression).ToList();
```

Comment:
I think it's confusing this var is called result. I expect a result after an Act step. Not an arrange step. Maybe name it `var mutations`?